app/views/posts/index.html.erb

```diff
@@ -1,22 +1,25 @@
-<div class="container-fluid">
-  <div class="row">
-    <%= render "topics/nav" %>
-  </div>
+<div class="col-md-3">
+  <%= render :partial => "topics/nav" %>
 </div>
-<% provide :title, t("posts.title") %>
-<% @posts.each do |post| %>
-  <div class="well">
-    <h1><%= post.title %></h1>
-    <p>
-      <h2><%= post.body %></h2>
-    </p>
-    <%= t "posts.created_at" %>
-    <%= link_to post.user.name, post.user %>
-    <%= time_ago_in_words post.created_at %>
-    <%= t "posts.ago" %><br>
-    <%= link_to t("read.more"), post_path(post), class: "btn btn-default" %>
+
+<div class="col-md-6">
+  <% provide :title, t("posts.title") %>
```

Comment:
You should write provide at the top of view

Reply:
done